app/javascript/components/visualization/controls/ScatterPlotLegend.jsx

```diff
@@ -60,16 +60,38 @@ function LegendEntry({
 
   // clicking the label will either hide the trace, or pop up a color picker
   const entryClickFunction = showColorControls ? () => setShowColorPicker(true) : toggleSelection
+
+  let scatterHoverHandle
+  let scatterHoverId
+  // do not trigger `setActiveTraceLabel()` immediately to prevent triggers on scroll
+  const scatterHoverHandler = (label, currentSelectCheck) => {
```

Comment:
Let's use classic function signatures, per previous review.  

It's consistent with other SCP JS, and enables easier "Lookup definition" when IDEs misbevave.